src/resources/third_party_externs.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @const
+ */
+var angular = {};
+
+/** @constructor */
+angular.Scope = function() {};
+
+/** @type {string} */
+angular.Scope.prototype.$$phase;
+
+/**
+ * @param {(string|function(!angular.Scope))=} opt_exp
+ * @return {*}
+ */
+angular.Scope.prototype.$apply = function(opt_exp) {};
+
+/**
+ * @param {(string|function(!angular.Scope))=} opt_exp
```

Comment:
what if one of these APIs mentions e.g. `Element`? I think it doesn't happen in the Angular externs, and even if should mostly just work, but might be worth thinking about and/or adding a test.